packages/app/src/app/components/Preview/DevTools/Terminal/index.tsx

```diff
@@ -34,6 +32,7 @@ class DevToolTerminal extends React.Component<
   };
 
   term: TerminalWithFit;
+  batchedMessages: any[];
```

Comment:
I'd name this `messageQueue` or `queuedMessages`?

Reply:
Ah... fixed, by suggesting code changes in Github directly... soooo, lazy 😂